Sources/ScipioKit/Producer/FrameworkProducer.swift

```diff
@@ -73,7 +73,8 @@ struct FrameworkProducer {
             assert(product.target.type == .library)
 
             let buildOptionsForProduct = buildOptions.overridingSDKs(for: product, platformMatrix: platformMatrix)
-            let compiler = Compiler(rootPackage: rootPackage, buildOptions: buildOptionsForProduct)
+//            let compiler = XcodeBuildCompiler(rootPackage: rootPackage, buildOptions: buildOptionsForProduct)
```

Comment:
`XcodeBuildCompiler` is the previous implementation. These sources still exist. However, not called anywhere.
I'll drop unnecessary implementations in a next PR